libraries/BufferedBleSpp/Xamarin Forms/BufferedBleSpp.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Plugin.BLE.Abstractions.Contracts;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace BufferedBleSppXamarin
+{
+    internal class BufferedBleSpp
+    {
+        // RX... and TX... naming is from server viewpoint, not the client (this code)
+
+        const int BUFFER_SIZE = 250;

```

Comment:
Best: have the constants with the proper naming convention. In this case: `const int BufferSize = 250;`